connect-four/connect4.js

```diff
@@ -0,0 +1,164 @@
+  // TODO: set "board" to empty HEIGHT x WIDTH matrix array
+  for(let i = 0; i < WIDTH; i++){
+    board[i] = [];
+    for(let j = 0; j < HEIGHT; j++){
+      board[i][j] = null;
+    }
+  }
+  return board;
+}
+
+/** makeHtmlBoard: make HTML table and row of column tops. */
+
+function makeHtmlBoard() {
+  // TODO: get "htmlBoard" variable from the item in HTML w/ID of "board"
+  const htmlBoard = document.querySelector("#board");
+  // TODO: add comment for this code
+  const top = document.createElement("tr"); //creating new table row
+  top.setAttribute("id", "column-top"); //setting the id attribute to 'column-top'
+  top.addEventListener("click", handleClick); //adding event listener to the tr to listen for a mouse click
+
+  for (let x = 0; x < WIDTH; x++) {
+    let headCell = document.createElement("td"); //creating td element
+    headCell.setAttribute("id", x); //set table data attribute id to 'x'
+    top.append(headCell); //append table data to trow
+  }
+  htmlBoard.append(top); //append table row to board
+
+  // TODO: add comment for this code: populate gameboard
+  for (let y = 0; y < HEIGHT; y++) {
+    const row = document.createElement("tr"); //creating table row element
+    for (var x = 0; x < WIDTH; x++) { 
+      const cell = document.createElement("td"); //creating table data element
+      cell.setAttribute("id", `${y}-${x}`); //giving td element an id attribute of "x"
+      row.append(cell); //append table data to table row 
+    }
+    htmlBoard.append(row); //append table row to board
+  }
+}
+
+//vvvv this comment made little sense vvv
+/** findSpotForCol: given column x, return top empty y (null if filled) */
+
+function findSpotForCol(x) {
+  // column(x how we move from left to right) stays the same as we loop through row (y how we move up and down) 
+  // TODO: write the real version of this, rather than always returning 0
+  for(let y = HEIGHT-1; y >= 0; y--){
+    if (board[x][y] === null){
+      board[x][y] = y;
+      return board[x][y];
+    }
+  }
+  return null;
+}
+
+/** placeInTable: update DOM to place piece into HTML table of board */
+
+function placeInTable(y, x) {
+  // TODO: make a div and insert into correct table cell
+  const newDiv = document.createElement("div")
+  newDiv.classList.add('piece');
+  newDiv.classList.add('p' + currPlayer);
+  const getTd = document.getElementById(`${y}-${x}`);
+  console.log(getTd);
+  getTd.append(newDiv);
+}
+
+/** endGame: announce game end */
+
+function endGame(msg) {
+  // TODO: pop up alert message
+  return alert(msg);
+}
+
+/** handleClick: handle click of column top to play piece */
+
+function handleClick(evt) {
+  // get x from ID of clicked cell
+  let x = +evt.target.id;
+
+  // get next spot in column (if none, ignore click)
+  let y = findSpotForCol(x);
+  if (y === null) {
+    return;
+  }
+
+  // place piece in board and add to HTML table
+  // TODO: add line to update in-memory board
+  placeInTable(y, x);
```

Comment:
Issue: I can't seem to get the colors of each players piece to show correctly on screen. The same color is being used for each piece no matter if I'm player1 or player2. I see the classes are being added to the div objects and the css classes match but only one style seems to display. Maybe I'm not updating correctly